src/TestClass.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: tony
+ * Date: 27.03.19
+ * Time: 14:49
+ */
+
+namespace Spatie\DataTransferObject;
+
+
+class TestClass implements immutable
```

Comment:
Can you explain exactly why this is needed?

Reply:
not i removed it